FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,26 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+
+			switch result {
+			case .success(let (data, response)):
+
+				guard response.statusCode == self.OK_200 else {
```

Comment:
Following from my previous comment, by making `OK_200` `static` we won't need to access it through `self`"
```suggestion
				guard response.statusCode == RemoteFeedLoader.OK_200 else {
```